### ISSUE TITLE
build -pack does not work

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#27720]
In master branch, in a clean repository, repro as follows:

``` none
cd Roslyn
restore
build -pack
```

![image](https://user-images.githubusercontent.com/3804346/41251846-0d794940-6d70-11e8-8674-fa170918645b.png)

This is blocking our attempt to diagnose #27523

**Comment by sharwell** [comment]
Related to #27235

Interim solution:

> .\\build\\scripts\\build.ps1 -build -pack

**Comment by gafter** [comment]
@sharwell That results in this dialog:

![image](https://user-images.githubusercontent.com/3804346/41259524-c2d1cb06-6d88-11e8-9379-839237ab69a2.png)

**Comment by gafter** [comment]
This worked:

> `powershell -noprofile -executionpolicy remotesigned -file .\build\scripts\build.ps1 -build -pack`

**Comment by sharwell** [comment]
@gafter yes, the command I wrote would only work from a PowerShell prompt. The alternative you added works from the legacy command prompt.
